apps/web/app/middleware/endpointValidator.ts

```diff
@@ -1,6 +1,6 @@
 export const loginRoute = (url: string) => url === "/api/auth/callback/credentials";
 
-export const signupRoute = (url: string) => url === "/api/v1/users";
+export const signupRoute = (url: string) => url.includes("/api/v1/users");
```

Comment:
naming-wise it's not a clean solution since it's named `signupRoute` but includes a lot of different routes. I think it would be a better solution to have different variables and handlers for forgotPassword, resetPassword and verifyEmail